ocaml/idl/datamodel_types.ml

```diff
@@ -26,12 +26,12 @@
 
 *)
 module Date = struct
-  open Xapi_stdext_date
+  module Date = Xapi_stdext_date.Date
   include Date
 
-  let iso8601_of_rpc rpc = Date.of_string (Rpc.string_of_rpc rpc)
+  let t_of_rpc rpc = Date.of_iso8601 (Rpc.string_of_rpc rpc)
 
-  let rpc_of_iso8601 date = Rpc.rpc_of_string (Date.to_string date)
+  let rpc_of_t date = Rpc.rpc_of_string (Date.to_rfc3339 date)
```

Comment:
This used to be `iso8601`, not `rfc3339`, and above it says `of_iso8601`. Not sure whether the distinction matters in practice, the doc comment for `rfc3339` says it is also iso8601 compatible.